README.md

```diff
@@ -34,7 +34,13 @@ Some popular ICO models:
 * [Simple Agreement for Future Tokens](https://medium.com/@argongroup/explaining-the-simple-agreement-for-future-tokens-framework-15d5e7543323)
 * [Cryptocurrency Airdrop](https://en.wikipedia.org/wiki/Airdrop_(cryptocurrency))
 
-TODO -- Fill in subcategories: security token, utility token, etc.
+Types of tokens:
+
+* [Utility Token](https://medium.com/coinmonks/utility-tokens-a-general-understanding-f6a5f9699cc0)
+* [Equity/Security Token](https://blockgeeks.com/guides/security-tokens/)
+* [Currency Token](https://cryptocurrencyfacts.com/what-is-a-cryptocurrency-token/)
```

Comment:
it'd be nice to have a small one paragraph description of what each of these is so that a viewer doesnt have to click over to the individual articles.